### PR TITLE
change eos mkdir

### PR DESCRIPTION
**Pull request by z4027163**
change mkdir in eos as suggested by eos team.
@sharad1126

### CODE REVIEW

**Comment by sharad1126** [comment]
@z4027163 the eos directory is `FINAL_HTML_DIR` only so let's change all the `cp` to `eos cp` and `mkdir` to the one you did but only for the ones where eos is concerned.

File: cWrap.sh [hunk 1]
```diff
@@ -10,8 +10,8 @@ fi
 cd $BASE_DIR
 
 modulename=`echo $1 | sed 's/\.py//' | sed 's/Unified\///'`
-mkdir -p $HTML_DIR/logs/$modulename/
-mkdir -p $FINAL_HTML_DIR/logs/$modulename/
+eos root://eoscms.cern.ch mkdir -p $HTML_DIR/logs/$modulename/
```
Comment (sharad1126) [inline cWrap.sh:13]: this shouldn't be eos as HTML_DIR is not eos directory!

**Comment by sharad1126** [comment]
@z4027163 can you please also check the `cp` to `eosp` in cWrap.sh script too wherever eos directory is used. it could help as well.

**Comment by z4027163** [comment]
@sharad1126 done

**Comment by sharad1126** [comment]
@z4027163  if we are sure about this PR that you added all eos infront of eos related cp and mkdir only, let's please merge this.

**Comment by sharad1126** [comment]
merging the PR. the shell check fails because of double quotes which will be resolved in #594

**Pull request merged**